core/block_svg.ts

```diff
@@ -268,7 +306,7 @@ export class BlockSvg
       const firstSibling: BlockSvg = surroundParent.getChildren(false)[0];
       const siblings: BlockSvg[] = [firstSibling];
       let nextSibling: BlockSvg | null = firstSibling;
-      while ((nextSibling = nextSibling.getNextBlock())) {
+      while ((nextSibling = nextSibling?.getNextBlock())) {
```

Comment:
Prevents error when dropping "random colour" or similar blocks into the workspace. Has unknown consequences, but not being able to do this was hindering testing.